netlib/utils.py

```diff
@@ -432,5 +432,26 @@ def safe_subn(pattern, repl, target, *args, **kwargs):
     """
     return re.subn(str(pattern), str(repl), target, *args, **kwargs)
 
-def bin_safe(s):
-    return ''.join(["\\x{:02x}".format(ord(i)) if ord(i) < 32 else i for i in s])
+
+def bytes_to_escaped_str(data):
+    """
+    Take bytes and return a safe string that can be displayed to the user.
+    """
+    if not isinstance(data, bytes):
+        raise ValueError("data must be bytes")
```

Comment:
They are **not** bytes. At least not in `dump.py` when they get printed (and wrapped with color etc.).